model/lifecycle.go

```diff
@@ -177,6 +178,83 @@ func AbortVersion(versionId, caller string) error {
 	return nil
 }
 
+func RemoveCommitQueueItem(projectId, item string) (bool, error) {
```

Comment:
maybe these functions should be in the model/commitqueue package?

Reply:
I think you're right, but because of dependency things no model functions can be called so I've rearranged a little.